site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -17,10 +17,56 @@
     <div class="upperinfo">
         <h1 class="upperinfo-left">New submission for: {{ gradeable_name }}</h1>
         {% if has_due_date %}
-            <h3 class="upperinfo-right">Due: {{ formatted_due_date }}</h3>
+            <h3 class="upperinfo-right">Due: {{ formatted_due_date }} <br>
+                {%  if due_in_a_day %}
+                    <span>Now: <span id="current-server-time">{{ formatted_current_time }}</span></span>
+                {% endif %}
+            </h3>
         {% endif %}
     </div>
 
+    <script>
+        $(function(){
+
+            // Get browser's time, timezone offset and unix timestamp (all in milliseconds)
+            var localDate = new Date();
+            var localTimestamp = localDate.getTime();
+            var localOffset = localDate.getTimezoneOffset() * 60000;
+
+            // Calculate shift to UTC in milliseconds
+            var UTCTime = localTimestamp + localOffset;
+            var serverTimeShift;
+
+            var fetchTimeUrl = window.location.href + "&action=fetch_time";
+
+            setTimeout(function(){
+                console.log("Ran");
```

Comment:
This `console.log` does not make sense to me. How about replacing it with a comment if something is needed here?

Reply:
Okay, that was there only for me to check if it was running properly at the correct time :D There were other logs too, overlooked this one while removing I guess :p